src/Compilers/CSharp/Test/Emit/Emit/DynamicAnalysis/DynamicInstrumentationTests.cs

```diff
@@ -2784,6 +2871,322 @@ static void Test()
             verifier.VerifyDiagnostics();
         }
 
+        [Fact]
+        public void TestSynthesizedConstructorWithSpansInMultipleFilesCoverage()
```

Comment:
I did try to use that - unfortunately it does not support the case where test code needs to be declared in multiple files and based on quick look it was non-trivial to add support for the multiple files case. I did not want to delay this change further by adding this support.